src/test/java/fi/otavanopisto/kuntaapi/test/utils/PtvDownload.java

```diff
@@ -0,0 +1,218 @@
+package fi.otavanopisto.kuntaapi.test.utils;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.FilenameFilter;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLConnection;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import org.apache.commons.lang3.StringUtils;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import fi.metatavu.ptv.client.model.V5VmOpenApiOrganizationService;
+import fi.metatavu.ptv.client.model.V7VmOpenApiOrganization;
+import fi.metatavu.ptv.client.model.V7VmOpenApiService;
+import fi.metatavu.ptv.client.model.V7VmOpenApiServiceServiceChannel;
+import fi.otavanopisto.kuntaapi.server.integrations.ptv.servicechannels.ServiceChannelType;
+
+public class PtvDownload {
+  
+  private static final String PTV = "v7";
+
+  public static void main(String[] args) throws JsonParseException, JsonMappingException, IOException {
+    new PtvDownload()
+      .downloadOrganizations()
+      .printIds();
+  }
+  
+  private void printIds() throws JsonParseException, JsonMappingException, IOException {
+    File[] organizationFiles = listJsonFiles(getOrganizationsFolder());
+    File[] serviceFiles = listJsonFiles(getServicesFolder());
+    File[] serviceChannelFiles = listJsonFiles(getServiceChannelsFolder());
+    
+    printIdList("ORGANIZATIONS", organizationFiles);
+    printIdList("SERVICES", serviceFiles);
+    printIdList("SERVICE_CHANNELS", serviceChannelFiles);
+    
+    for (ServiceChannelType type : ServiceChannelType.values()) {
+      printServiceChannelList(type, serviceChannelFiles);
+    }
+  }
+  
+  private ServiceChannelType resolveChannelType(String type) {
+    switch (type) {
+      case "EChannel":
+        return ServiceChannelType.ELECTRONIC_CHANNEL;
+      case "ServiceLocation":
+        return ServiceChannelType.SERVICE_LOCATION;
+      case "PrintableForm":
+        return ServiceChannelType.PRINTABLE_FORM;
+      case "Phone":
+        return ServiceChannelType.PHONE;
+      case "WebPage":
+        return ServiceChannelType.WEB_PAGE;
+      default:
+        return null;
+    }
+  }
+
+  private void printIdList(String constName, File[] files) {
+    List<String> ids = new ArrayList<>();
+    for (File file : files) {
+      ids.add(String.format("      \"%s\"",  StringUtils.stripEnd(file.getName(), ".json")));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal ".json" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)